### PR TITLE
List: Add `scrollToIndex` method to `IList` interface

### PR DESCRIPTION
**Pull request by cschleiden**
List: Add `scrollToIndex` method to `IList` interface

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/List/List.Props.ts [hunk 1]
```diff
@@ -3,7 +3,17 @@ import { IRectangle } from '../../Utilities';
 import { List } from './List';
 
 export interface IList {
-
+  /**
+   * Scroll to the given index. By default will bring the page the specified item is on into the view. If a callback
+   * to measure the height of an individual item is specified, will only scroll to bring the specific item into view.
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/List/List.Props.ts]: Is there an example of this being used in an example?

Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/List/List.Props.ts]: Yes, the "Scrolling items into view" example: http://dev.office.com/fabric#/components/list

Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/List/List.Props.ts]: Ok, so this function was usable in the past, just wasn't specified in an interface.

Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/List/List.Props.ts]: @micahgodbolt Yep, it predates the `componentRef` work

**Pull request merged**